Portfolio Website: Added Menu to Mobile Website and Implemented Toggle Menu

rmiyoyo opened the pull request:
# Work Done
- Added a menu to the mobile version of the portfolio website.
- Used JavaScript to effect the toggle menu functionality.
- Added JavaScript linters and fixed errors.
- Updated README to include JavaScript in the tech stack.
- Added links to the menu buttons.

rmiyoyo committed:
Create new branch and add JavaScript Linter

rmiyoyo committed:
Start adding JS functionalities

momo-87 committed:
Mobile menu class style defined and updated

momo-87 committed:
JavaScrit added to the lis of used technologies in the README.md file

rmiyoyo committed:
Add mobile menu and the x button

rmiyoyo committed:
Style the mobile menu

rmiyoyo committed:
Implement the menu toggle feature for the mobile website.

rmiyoyo committed:
Add the X button image to the images folder

rmiyoyo committed:
Update linters.yml to include eslint

AbelG101 commented on style.css:
```diff
@@ -1,3 +1,51 @@
+  position: fixed;
+}
+
+.mobile-menu {
+  visibility: visible;
+  padding: 5%;
+  position: fixed;
+  left: 0;
+  top: 0;
+  overflow: hidden;
+  max-width: 767px;
+  min-width: 375px;
+  width: 100%;
+  height: 600px;
+  background-color: #6070ff;
+  opacity: 0.9;
+  z-index: 9;
+  transition: opacity 0.5s, visibility 0.5s;
+}
```
- [ ] Ouch! this is so close but it seems like the `height: 600px;` you've added on the sidebar is preventing it from filling the entire screen as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=cd5TWUpyWfE6T9vR-0)🎨. So please make sure to give it a `100vh` height 🎯.
- [ ] There seems to be a background blur happening as shown in the design so please make sure to implement it:
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/231214474-802422ad-a906-49ae-a26e-545bedcc0919.png)
    - It could be a bit challenging but you can find out how it works after doing some research. 💪

AbelG101 reviewed (changes requested):
![](https://media.giphy.com/media/ReyD4OdRiHeLDCRaEP/giphy.gif)


# Hi @rmiyoyo & @momo-87  👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rmiyoyo committed:
Add blur effect to body when menu is active.

rmiyoyo committed:
Fix linter errors

rmiyoyo commented:
# Work Done After First Review by @AbelG101 
- Added the blur effect to the body when the mobile menu is active.
- Fixed the height of the mobile menu to match.

Shisui6 reviewed (approved):
Hi @rmiyoyo and @momo-87, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Mobile menu works perfectly ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
